docs/storage.md

```diff
@@ -316,6 +322,39 @@ Now you can create the SR where `server` is your mon ip.
 * Do not use admin keyring for production, but make a separate key with only necessary privileges https://docs.ceph.com/en/latest/rados/operations/user-management/
 :::
 
+### MooseFS
+
+Shared, thin-provisioned storage. Available since XCP-ng 8.2.
+
+MooseFS is a fault-tolerant, highly available, highly performing, scaling-out, network distributed file system.  It is POSIX compliant and acts like any other Unix-like file system.
+SR driver was contributed directly by MooseFS Developement Team.
+
+:::warning
+The MooseFS client is not included with XCP-ng, so it must be installed on dom0 from the official MooseFS repository.
+:::
+
+Installation steps
+```
+curl "https://ppa.moosefs.com/RPM-GPG-KEY-MooseFS" > /etc/pki/rpm-gpg/RPM-GPG-KEY-MooseFS
+curl "http://ppa.moosefs.com/MooseFS-3-el7.repo" > /etc/yum.repos.d/MooseFS.repo
+yum install moosefs-client
+```
```

Comment:
In https://xcp-ng.org/docs/additionalpackages.html we tell users to avoid leaving additional repos enabled, and advise instead to use yum's `--enablerepo` when necessary. This is necessary for large generic repos such as CentOS base or EPEL, but in your case leaving the repo enabled may be less problematic.

However, leaving the repo enabled means that any update you do to the packages will be offered to our users as an update, so I think you should at least mention this fact, and tell them to disable the repo and use `--enablerepo` when they specifically want to update MooseFS packages.